config/locales/en.yml

```diff
@@ -497,8 +497,8 @@ en:
     new:
       title: Create your job alert
     unsubscribe:
-      confirmation: You will no longer receive updates for this alert.
-      header: You have been unsubscribed from this email
+      confirmation: You will no longer receive updates for this job alert.
+      header: You have been unsubscribed from this alert
```

Comment:
Let's change this text either in both places, or in neither (see above screenshot of creating a subscription for NQTs/creating one normally). Also, please run it past Content Design.